gox/plan.sh

```diff
@@ -0,0 +1,24 @@
+pkg_origin=core
+pkg_name=gox
+pkg_version=master
+pkg_maintainer="The Habitat Maintainers <humans@habitat.sh>"
+pkg_license=("Apache 2")
+pkg_source=https://github.com/mitchellh/gox/archive/master.tar.gz # I'm killing a kitten.
+pkg_upstream=https://github.com/mitchellh/gox
```

Comment:
pkg_upstream => pkg_upstream_url

Travis should have caught this - @eeyun travis should run pre-commit against any PR, right?